README.md

```diff
@@ -52,18 +52,27 @@ NEXT_PUBLIC_AWS_SECRET_ACCESS_KEY=...
 NEXT_PUBLIC_AWS_REGION=...
 NEXT_PUBLIC_EMAIL=...
 
+  <img src="https://raw.githubusercontent.com/net-partner-011-ab/code-crafter/ON-25/update-read-me/assets/img/Screenshot%202024-01-17%20at%2009.26.09.png" width="400">
```

Comment:
Image tags are added inside of the previous element:
<img width="859" alt="Screenshot 2024-01-20 at 06 54 57" src="https://github.com/net-partner-011-ab/code-crafter/assets/49514404/2f79b26d-fd49-4a69-8823-ad95e00733ae">